### PR TITLE
Fixed some issues with PR #14157

### PR DESCRIPTION
**Pull request by ariel-**
Feel free to destroy this one too, I'm open to suggestions. :grinning:

### CODE REVIEW

File: src/server/scripts/Outland/HellfireCitadel/ShatteredHalls/shattered_halls.h [hunk 1]
```diff
@@ -77,7 +95,7 @@ struct FactionSpawnerHelper
     FactionSpawnerHelper(uint32 allianceEntry, uint32 hordeEntry, const Position& pos) : _allianceNPC(allianceEntry), _hordeNPC(hordeEntry), _spawnPos(pos) { }
 
     inline uint32 operator()(uint32 teamID) const { return teamID == ALLIANCE ? _allianceNPC : _hordeNPC; }
-    inline const Position GetPos() const { return _spawnPos; }
+    inline Position GetPos() const { return _spawnPos; }
```
Comment (jackpoz) [inline src/server/scripts/Outland/HellfireCitadel/ShatteredHalls/shattered_halls.h]: what do you need this change for ?

**Comment by ariel-** [comment]
It was unnecessary, since the function is returning a Position by copy, marking it as 'const' does nothing.

**Comment by Naios** [comment]
Isn't there a `::GetHomePosition()`?.
Anyway probably its better to return `Position` as constant reference.

**Comment by Aokromes** [comment]
We can’t automatically merge this pull request.
Use the command line to resolve conflicts before continuing.

**Pull request merged**